content/pt-br/community/code-of-conduct.md

```diff
@@ -1,27 +1,30 @@
 ---
-title: Community
+title: Código de Conduta da Comunidade Kubernetes
 layout: basic
 cid: community
-css: /css/community.css
+community_styles_migrated: true
 ---
 
-<div class="community_main">
-<h1>Código de Conduta da Comunidade Kubernetes</h1>
-
+<div class="community-section" id="cncf-code-of-conduct-intro">
+<p>
 Kubernetes segue
-<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Código de Conduta</a>.
+<a href="https://github.com/cncf/foundation/blob/main/code-of-conduct.md">CNCF Código de Conduta</a>.
 O texto do CNC CoC é reproduzido abaixo, a partir de
-<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+<a href="https://github.com/cncf/foundation/blob/fff715fb000ba4d7422684eca1d50d80676be254/code-of-conduct.md">commit fff715fb0</a>.
```

Comment:
We have the code-of-conduct.md in portuguese. Would you please update it.
https://github.com/cncf/foundation/blob/fff715fb000ba4d7422684eca1d50d80676be254/code-of-conduct-languages/pt.md

Reply:
Yes! I have an open PR for it: https://github.com/cncf/foundation/pull/480